src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -263,6 +270,14 @@ public void handleHome() {
     }
 
     /**
+     * Opens the birthday statistics panel.
+     */
+    @FXML
+    public void handleBirthdayStatistics() {
```

Comment:
Remove this, unless you have added a menu item to open birthday statistics. If you did then do "force" testings on this under ChangeInformationPanelSystemTest, refer to how I did the testings in that system test for handleHome().